spikeinterface/extractors/phykilosortextractors.py

```diff
@@ -48,7 +48,12 @@ def __init__(self, folder_path, exclude_cluster_groups=None, keep_good_only=Fals
         # try to load cluster info
         cluster_info_files = [p for p in phy_folder.iterdir() if p.suffix in ['.csv', '.tsv']
                               and "cluster_info" in p.name]
-        if len(cluster_info_files) == 1:
+
+        if len(cluster_info_files) == 0:
```

Comment:
This is strange. Why in a particular case this file is not present ?
Is it version related ?

If not exists I am not sure the `unsorted` is the good naming by default.

Reply:
@samuelgarcia yes this is for ks1 and unsorted is the default name